### PR TITLE
Capture smartsheet exception more granularly && Add sentry service to capture exception or message

### PR DESCRIPTION
**Pull request by zhx828**
This is related to https://github.com/oncokb/oncokb-pipeline/issues/231

### CODE REVIEW

File: src/main/java/org/mskcc/cbio/oncokb/service/SentryService.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        }
+        Sentry.captureMessage(message);
+    }
+
+    /**
+     * Send the exception to Sentry
+     *
+     * @param sentryLevel Sentry event level. Warning, Error and more
+     * @param exception   Any throwable exception which will be displayed in the Sentry issue
+     * @param message     he custom message which will be displayed in the Sentry issue
```
Comment (calvinlu3) [inline src/main/java/org/mskcc/cbio/oncokb/service/SentryService.java:33]:
```suggestion
     * @param message     The custom message which will be displayed in the Sentry issue
```

**Review by calvinlu3 — APPROVED** [review]
Looks good, but I have one question.

Is there a way to send additional info other than the user information to sentry?

**Review by bprize15 — APPROVED** [review]
LGTM

**Comment by zhx828** [comment]
> Looks good, but I have one question.
> 
> Is there a way to send additional info other than the user information to sentry?
Yeah, User is a model the Sentry supports out of the box. You can be creative to support additional messages. You can use breadcrumbs, extra or tag.

**Pull request merged**